<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="me.dio.credit.application.system.service.CustomerServiceTest" tests="4" skipped="0" failures="0" errors="0" timestamp="2023-11-19T19:59:06" hostname="TIAGO" time="0.351">
  <properties/>
  <testcase name="should not find customer by invalid id and throw BusinessException()" classname="me.dio.credit.application.system.service.CustomerServiceTest" time="0.202"/>
  <testcase name="should find customer by id()" classname="me.dio.credit.application.system.service.CustomerServiceTest" time="0.009"/>
  <testcase name="should delete customer by id()" classname="me.dio.credit.application.system.service.CustomerServiceTest" time="0.116"/>
  <testcase name="should create customer()" classname="me.dio.credit.application.system.service.CustomerServiceTest" time="0.012"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
